app/search.py

```diff
@@ -139,6 +139,7 @@ def getVenueDetailsFromProvider(namespace, idObj, cached):
         
     resolve = resolvers[namespace]
     try:
+        # NB: This is updated to handle ids directly, rather than parsing from an object
```

Comment:
I'm not sure what this means for a developer reading this code - can you elaborate?

Does this mean the function needs to be updated? (if so, add a TODO and issue). Does this mean the function just uses IDs now? (if so, update the variable names so it's clear and remove the comment).